src/labs/requests/NewLabRequest.tsx

```diff
@@ -83,27 +97,60 @@ const NewLabRequest = () => {
     }
   }
 
+  const onVisitChange = (value: string) => {
+    setNewLabRequest((previousNewLabRequest) => ({
+      ...previousNewLabRequest,
+      visitId: value,
+    }))
+  }
+
   const onCancel = () => {
     history.push('/labs')
   }
 
+  const defaultSelectedVisitsOption = () => {
```

Comment:
Maybe a good idea to add a test for this functionality

Reply:
Great idea. I added a test that chooses a patient in the Typeahead, makes sure the option isn't selected yet, and then tests selecting the option, which all leverages this `defaultSelectedVisitsOption` function.